### ISSUE TITLE
Mac key remapping not sticking

### ISSUE THREAD

**Issue by jimpriest** [dbeaver/dbeaver#8546]
#### System information: 
- MacOS 10.14.6
- DBeaver version: 7.0.01

#### Connection specification:
- MSSQL

#### Describe your question:
I'm a Windows user on a Mac and am trying to remap my Home and End keys in Preferences > Keys for Line Start/Line End to Home and End but these settings don't seem to persist?  

I've tried both Line and Text.

**Comment by uslss** [comment]
hi @jimpriest ,
unfortunately, cannot reproduce an issue. Did changes apply after application restart?

**Comment by serge-rider** [comment]
Seems to be duplicate of #4483

**Comment by jgonzale** [comment]
+1 with this issue.

In DBEaver 7.2, macOS 10.15.7.

Even remapping the Line Start to Home / Line End to End actions in the Text edition context, the Home and End button move the cursor to **the beginning and end of the whole editor window**.

Pretty annoying 😢

**Comment by Rah1x** [comment]
still happening!

**Comment by cardalda** [comment]
And still...
It happens very quickly after a restart without any clue.
I have been pressing END after every query, and for no particular reason after executing a few queries, END jumps to end of file instead end of line.
After this, I cannot even save my file sometimes, neither with keyboard, nor by the menu bar.

I think I found something.
It has nothing to do with executing queries.
I can (almost) reproduce it by:

I was editing the following query

    select ag.*
         , cm.name, cm.image
         , ev.msg_content
      from (
        select camp_id
             , max(id)   as last_id --➜ obtain last event 
             , count(*)  as total
             , ***
         from events
        where target_id = 2
        group by camp_id
     ) ag
      join events    ev on ev.id = ag.last_id
 left join campaigns cm on cm.id = ag.camp_id
;

At the *** mark I started typing (Just restarted dBeaver and waited until the editor has made it's connection tot the database)

case when seen_at is null then 1 e

when pressing the e from else, sometimes I am too slow and the autocomplete wants to popup; sometimes, not always, the key mapping is gone.
I guess this is some kind of race condition with the autocompleter..

I have tried it multiple times, and I succeeded in loosing my key mappings three times, within 5 seconds after opening dBeaver...
